test/extended/util/annotate/generated/zz_generated.annotations.go

```diff
@@ -1831,11 +1831,11 @@ var Annotations = map[string]string{
 
 	"[sig-etcd] etcd record the start revision of the etcd-operator [Early]": " [Suite:openshift/conformance/parallel]",
 
-	"[sig-etcd][Feature:DisasterRecovery][Disruptive] [Feature:EtcdRecovery] Cluster should recover from a backup taken on one node and recovered on another [apigroup:operator.openshift.io]": " [Serial]",
+	"[sig-etcd][Feature:DisasterRecovery][Disruptive][Disabled:Broken] [Feature:EtcdRecovery] Cluster should recover from a backup taken on one node and recovered on another [apigroup:operator.openshift.io]": " [Serial]",
 
-	"[sig-etcd][Feature:DisasterRecovery][Disruptive] [Feature:EtcdRecovery] Cluster should restore itself after quorum loss [apigroup:machine.openshift.io][apigroup:operator.openshift.io]": " [Serial]",
+	"[sig-etcd][Feature:DisasterRecovery][Disruptive][Disabled:Broken] [Feature:EtcdRecovery] Cluster should restore itself after quorum loss [apigroup:machine.openshift.io][apigroup:operator.openshift.io]": " [Serial]",
 
-	"[sig-etcd][Feature:DisasterRecovery][Suite:openshift/etcd/recovery] [Feature:EtcdRecovery] should install ssh keys on CP nodes": "",
+	"[sig-etcd][Feature:DisasterRecovery][Suite:openshift/etcd/recovery] [Feature:EtcdRecovery][Disruptive] Restore snapshot from node on another single unhealthy node": " [Serial]",
```

Comment:
Actually, I just noticed we have the `[Serial]` tag appended here now. Not sure why it's getting added here but hopefully this doesn't mean it's going to run as part of the serial suite.

Reply:
@dgoodwin do you happen to know how this ends up here? Is it due to the disruptive tag? 

If it's OK then could you also /approve? I think we got nobody around that has approval rights on the generated file (Michal and David are out during Kubecon).

Reply:
Apologies I am not sure if [serial] is from [disruptive] or not, but it seems likely, you will need to check. It seems logical that disruptive tests may require running in serial.

Could you send me a link to a presubmit that ran the test? I can't seem to find it in any.

Depending on how many jobs are expected to run this test, you should probably run some payload commands to make sure this isn't going to unleash havoc once the jira requirement to merge lifts. (payload 4.14 nightly blocking, and payload 4.14 ci blocking)

Reply:
Thanks @dgoodwin. Test-presubmit is here: https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_cluster-etcd-operator/1041/pull-ci-openshift-cluster-etcd-operator-master-e2e-aws-etcd-recovery/1649350721021153280

as you can see, we disable both of the failing tests with this PR. So should get better and not worse ;-)

We only added it here: https://github.com/openshift/release/pull/37799 for etcd and the CEO repo, so shouldn't impact any payload AFAIU.

Reply:
Ah awesome, an aws-etcd-recovery job is perfect, ty!